backend/src/main/java/matches/organizer/controller/MatchController.java

```diff
@@ -33,4 +31,14 @@ public MatchController(MatchService matchService) {
     public @ResponseBody List<MatchDTO> getAllMatches() {
         return matchService.getMatches().stream().map(match -> match.getDto()).collect(Collectors.toList());
     }
+
+    @PostMapping(value = "/matches", produces = MediaType.APPLICATION_JSON_VALUE, consumes = MediaType.APPLICATION_JSON_VALUE)
+    public String createMatch(@RequestBody POSTMatchDTO oneMatch){
+        return "Partido creado con ID: " + matchService.createMatchFromPost(oneMatch.getName(),oneMatch.getLocation(),oneMatch.getDate(),oneMatch.getHour());
```

Comment:
La response esta especificada como JSON, pero se esta devolviendo un string.

Creeria que la respuesta mas indicada para el POST, es un JSON que contenga el objeto creado (Seria un return con el Objeto Java).

Las razones para devolver el objeto creado en el POST es una buena practica para obtener la nueva representacion del estado del objeto.
Adicionalmente, el status code deberia ser 201 (Created)

Dejo mas informacion sobre esto en los siguientes Links:
- https://www.vinaysahni.com/best-practices-for-a-pragmatic-restful-api#useful-post-responses
- https://stackoverflow.com/questions/19199872/best-practice-for-restful-post-response